### PR TITLE
pkp/pkp-lib#3691 galley styles

### PR DESCRIPTION
**Pull request by Vitaliy-1**

### CODE REVIEW

**Comment by Vitaliy-1** [comment]
It seems that all checks are passed. Some comments: I have tried several approaches, in my opinion injecting a reference to CSS is better to be done through intercepting HTML galley string than accessing iframe with JS. usage: 
`(ThemePlugin child)->addStyle("htmlStyle", "default.css", array('contexts' => 'htmlGalley'));`
by analogy with this removing style.

**Comment by Vitaliy-1** [comment]
And, probably, 2 letters should be switched in place in the method name; sorry, misspelled

**Comment by mfelczak** [comment]
Thanks @Vitaliy-1! I'll wait until this has been reviewed.

File: plugins/generic/htmlArticleGalley/HtmlArticleGalleyPlugin.inc.php [hunk 1]
```diff
@@ -180,6 +180,8 @@ function _getHTMLContents($request, $galley) {
 		foreach ($paramArray as $key => $value) {
 			$contents = str_replace('{$' . $key . '}', $value, $contents);
 		}
+		
+		HookRegistry::call('HtmlArticleGalleyPlugin::htmlGalleyContent', array(&$galley, &$contents));
```
Comment (NateWr) [inline plugins/generic/htmlArticleGalley/HtmlArticleGalleyPlugin.inc.php]: Let's pass the `$submissionFile` and `$embeddableFiles` and `$issue` with the hook. That way, we can save extra hits to the database to retrieve data that's already been retrieved.

**Comment by mfelczak** [comment]
Hi @Vitaliy-1 and @NateWr, is this one ready to go (other than completing the merge)?

**Comment by Vitaliy-1** [comment]
Yes, it's ready.

**Comment by NateWr** [comment]
https://github.com/pkp/ojs/pull/2228

**Comment by mfelczak** [comment]
Thanks! For some reason I didn't get the recent github notifications on this one but I suspect that's my mail filters ...

**Pull request closed without merging**